### PR TITLE
[CloudMinds] 10331-Robot Service Info

### PR DESCRIPTION
**Pull request by Kenny-SZ**
This LWM2M Object includes robot service related information.

### CODE REVIEW

File: 10331.xml [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+        <Description><![CDATA[0:Moving, 1:Stopped, 2:Suspended.]]></Description>
+      </Item>
+      <Item ID="7">
+        <Name>Moving Speed</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>M/H</Units>
```
Comment (seanmcilroy29) [inline 10331.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

Comment (Kenny-SZ) [reply 10331.xml]: M/H is the unit of speed, how much meters per hour.

File: 10331.xml [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+        <Description><![CDATA[Current language used by the robot.]]></Description>
+      </Item>
+      <Item ID="5">
+        <Name>Volume</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>String</Type>
+        <RangeEnumeration>0..100</RangeEnumeration>
+        <Units>%</Units>
```
Comment (seanmcilroy29) [inline 10331.xml]: % should be updated to /100

Comment (Kenny-SZ) [reply 10331.xml]: Changed.

File: 10331.xml [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+        <Description><![CDATA[0:Moving, 1:Stopped, 2:Suspended.]]></Description>
+      </Item>
+      <Item ID="7">
+        <Name>Moving Speed</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>m/h</Units>
```
Comment (cabo) [inline 10331.xml:112]: You have rather slow robots if m/h is a natural unit.
Why not km/h (or m/s)?

**Pull request merged**